apps/condo/domains/settings/utils/MobileFeatureConfigSchemaValidations.js

```diff
@@ -1,9 +1,11 @@
+const { get } = require('lodash')
+
 const { GQLError } = require('@open-condo/keystone/errors')
 
 const { normalizePhone } = require('@condo/domains/common/utils/phone')
 
-const ticketSubmittingValidations = async (resolvedData, context, ERRORS) => {
-    if (resolvedData.ticketSubmittingIsDisabled) {
+const ticketSubmittingValidations = async (resolvedData, context, existingItem, ERRORS) => {
+    if (resolvedData.ticketSubmittingIsDisabled || get(existingItem, 'ticketSubmittingIsDisabled')) {
         if (!resolvedData['commonPhone']) {
```

Comment:
In case, if you have already specified a phone without `ticketSubmittingIsDisabled` flag and then you specify the flag, then there will be an error.

Reply:
you can write something like this `const newItem = { ...existingItem, ...resolvedData }` and use `newItem` in all conditions